src/libraries/Microsoft.PowerFx.Interpreter/Functions/FileInfoImpl.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Interpreter;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx
+{
+    internal class FileInfoFunctionImpl : FileInfoFunction, IAsyncTexlFunction5
```

Comment:
>IAsyncTexlFunction5 [](http://example.com/codeflow?start=60&length=19)

Can you implement `IAsyncTexlFunction3` instead?

Reply:
I think so